warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -423,20 +424,25 @@ public int run(String[] args) throws Exception {
         // output the counters to the log
         Counters counters = job.getCounters();
         log.info(counters);
+        JobClient jobClient = new JobClient((org.apache.hadoop.mapred.JobConf) job.getConfiguration());
```

Comment:
We still need to close this jobClient which is what the try block was doing for us previously.

Reply:
true!